proposals/exception-handling/Exceptions.md

```diff
@@ -258,6 +258,18 @@ end
 The `rethrow` here references `try $l2`, but the `rethrow` is not within its
 `catch` block.
 
+Also, targetting a label of `catch`-less `try` or `try`-`delegate` is also a
```

Comment:
I still don't see the relevance of the "catch-less" here, I think it's misleading: the example is a validation failure regardless of whether you add a catch clause or not. Both examples are invalid simply because rethrow does not reference a catch-block, no other reason.

Reply:
I guess the whole added paragraph here about `rethrow` is not really meaningful and also misleading as you suggested. This PR's goal was to clarify about catchless try and its relationship w/ `delegate`, so I'll delete this whole paragraph and example.